src/Modules/Data/Commands.Data/help/Remove-PowerBIRows.md

```diff
@@ -0,0 +1,130 @@
+---
+external help file: Microsoft.PowerBI.Commands.Data.dll-Help.xml
+Module Name: MicrosoftPowerBIMgmt.Data
+online version:
+schema: 2.0.0
+---
+
+# Remove-PowerBIRows
+
+## SYNOPSIS
+Removes all rows from the Power BI table.
+
+## SYNTAX
+
+### Dataset (Default)
+```
+Remove-PowerBIRows -Dataset <Dataset> -TableName <String> [-WorkspaceId <Guid>] [-Workspace <Workspace>]
+ [<CommonParameters>]
+```
+
+### DatasetId
+```
+Remove-PowerBIRows -DatasetId <Guid> -TableName <String> [-WorkspaceId <Guid>] [-Workspace <Workspace>]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+The Remove-PowerBIRows cmdlet lets you remove rows from the Power BI table. 
```

Comment:
Allows removal of rows from the Power BI table.

Reply:
Done